graphstorm-processing/graphstorm_processing/repartition_files.py

```diff
@@ -135,17 +142,45 @@ def write_parquet_to_relative_path(
 
     @staticmethod
     def create_new_relative_path_from_existing(
-        original_relative_path: str, repartitioned_file_index: int
+        original_relative_path: str, repartitioned_file_index: int, suffix: str = None
     ) -> str:
-        """
+        """Changes the index in the `original_relative_path` to `original_relative_path`.
```

Comment:
What does this mean?

Reply:
I'll need to fix this. What I wanted to say is that we change the file index indicator in `original_relative_path`, which will be something like `original/path/part-00003.parquet` to `original/path/part-00001.parquet` if `repartitioned_file_index` was 1.

Reply:
I c.
But can you extend the docstring to include an example like what you commented here?